src/synapse/azext_synapse/_help.py

```diff
@@ -191,8 +207,7 @@
 examples:
   - name: Pause SQL pool.
     text: |-
-        az synapse sql pool pause --name sqlpoolcli1 --resource-group rg --workspace-name \\

```

Comment:
I suppose the long summary is not accurate now. Remove it?

Reply:
fixed. I will remove all the long summary.

---
In reply to: [416229828](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r416229828) [](ancestors = 416229828)